benchmark/clblast/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.2.2)
 
+include(FindCLBlast)
```

Comment:
Find Module should not be included. Just `find_package` them

Reply:
Fixed